plugin/storage/es/mappings/mapping_test.go

```diff
@@ -174,12 +203,18 @@ func TestMappingBuilderGetSpanServiceMappings(t *testing.T) {
 		{
 			name: "ES Version 7",
 			args: args{
-				shards:        3,
-				replicas:      3,
-				esVersion:     7,
-				indexPrefix:   "test",
-				useILM:        true,
-				ilmPolicyName: "jaeger-test-policy",
+				ShardsSpan:           3,
+				ReplicasSpan:         3,
+				ShardsService:        3,
+				ReplicasService:      3,
+				ShardsDependencies:   3,
+				ReplicasDependencies: 3,
+				ShardsSampling:       3,
+				ReplicasSampling:     3,
```

Comment:
it looks like these values are always the same across all test cases - if so there is no need to declare them as part of the test case, just initialize them inside `t.Run` as needed. Test case struct is supposed to only contain parameters that are actually varied across the tests.